satviz-consumer/src/main/java/edu/kit/satviz/consumer/config/jsonparsing/ColorSerializer.java

```diff
@@ -0,0 +1,32 @@
+package edu.kit.satviz.consumer.config.jsonparsing;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import java.io.IOException;
+import javafx.scene.paint.Color;
+
+/**
+ * This class is responsible for the serialization of an instance of the
+ * {@link javafx.scene.paint.Color} class into a hex representation.
+ */
+public class ColorSerializer extends JsonSerializer<Color> {
+
+  @Override
+  public void serialize(Color value, JsonGenerator gen, SerializerProvider sp) throws IOException {
+    gen.writeString("#" + addPrecedingZeroes(Integer.toHexString(value.hashCode())));
```

Comment:
Die eigene Methode `addPrecedingZeros` ist überflüssig.
```suggestion
    gen.writeString("#" + value.hashCode().toString("X6"));
```

Reply:
Das war ein ziemlich guter Tipp denke ich.